arch/arm/arm64/tls.c

```diff
@@ -86,7 +102,7 @@ extern char _tls_start[], _etdata[], _tls_end[];
  *             | Custom TCB format       | |   > Thread Control Block (TCB)
  *             | (might be used          | |  |  (length: TCB_SIZE)
  *             |  by a libC)             | |  |
- *       tlsp -+-> AARCH64_RESERVED      | |  |
+ *       tlsp -+-> AARCH64_TCB_OVERLAP   | |  |
```

Comment:
There is another note/reference to the overlap under this diagram that needs to be updated / deleted. Maybe the explanation next to `AARCH64_RESERVED` is enough.

Reply:
I've tried to come up with an update. I didn't want to delete it because I found your original comment to be very useful in understanding the TLS layout. Please let me know if you find it okay :)